package-lock.json

```diff
@@ -26151,13 +26153,18 @@
         "react-dom": "^17.x"
       }
     },
+    "packages/console/node_modules/@deephaven/jsapi-bootstrap": {
+      "resolved": "packages/bootstrap",
```

Comment:
This is still pointing to the wrong directory for some reason... Maybe revert all changes to the package-lock, and just run `npm install` again ? E.g.:
```
git co main -- package-lock.json
npm install
```